pkg/web/mobileResourceHandler.go

```diff
@@ -0,0 +1,54 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/aerogear/mobile-client-service/pkg/mobile"
+	"github.com/gorilla/websocket"
+	"github.com/labstack/echo"
+)
+
+var upgrader = websocket.Upgrader{
+	ReadBufferSize:  1024,
+	WriteBufferSize: 1024,
+}
+
+// MobileResourceHandler handles queries for mobile client custom resources.
+type MobileResourceHandler struct {
+	lister mobile.MobileResourceLister
+}
+
+// NewMobileResourceHandler returns a MobileResourceHandler
+func NewMobileResourceHandler(lister mobile.MobileResourceLister) *MobileResourceHandler {
+	return &MobileResourceHandler{
+		lister: lister,
+	}
+}
+
+func (mrh *MobileResourceHandler) list(context echo.Context) error {
+	mobileClientResourceList, err := mrh.lister.List()
+	if err != nil {
+		return err
+	}
+
+	return context.JSON(http.StatusOK, mobileClientResourceList)
+}
+
+func (mrh *MobileResourceHandler) watch(context echo.Context, resourceVersion string) error {
```

Comment:
looks like this doesn't do anything. Is it right?

Reply:
Yeah it's not needed currently but we will need watch handling in the future. I can remove it until we need it if you want?

Reply:
no, it's ok. We can leave it here and it will remind us what's required.